sellotape/sellotape_dj/sellotape_dj/settings.py

```diff
@@ -0,0 +1,124 @@
+"""
+Django settings for sellotape_dj project.
+
+Generated by 'django-admin startproject' using Django 3.1.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/3.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/3.1/ref/settings/
+"""
+
+from pathlib import Path
+
+# Build paths inside the project like this: BASE_DIR / 'subdir'.
+BASE_DIR = Path(__file__).resolve(strict=True).parent.parent
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.1/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = 'gtk7i_g+v%afq6dvplwms_-*r3j7uj-k6g4b&&iu#ys-e(ui)0'
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = [
+    '0.0.0.0',
+    '127.0.0.1',
+]
```

Comment:
While running the server on the guest machine, accessing it through the host machine can often get an `ERR_CONNECTION_REFUSED`. Seems like this is caused by the server within the VM because it binds to `127.0.0.1`, which is loopback. We want to make sure that the server is bound to `0.0.0.0` so that all interfaces can access it.

https://stackoverflow.com/questions/18157353/connection-reset-when-port-forwarding-with-vagrant